webauthn/webauthn.py

```diff
@@ -1119,7 +1128,6 @@ def _encode_public_key(public_key):
     return b'\x04' + binascii.unhexlify('{:064x}{:064x}'.format(
         numbers.x, numbers.y))
 
-
```

Comment:
you have a newline lost here and in the test file, probably want to add them back to make merging back into the mainline easier